### ISSUE TITLE
Counter Attempt to Format the desc and unit Arguments

### ISSUE THREAD

**Issue by ZivRonen** [Rockhopper-Technologies/enlighten#45]
**Describe the bug**
Enlighten Counter try to format the description/unit fields  - it will cause issues if '{' or '}' are present, in most cases raising ValueError.
This happen in both progress bar and counter modes.

**To Reproduce**

```python
"""
Progress bar
"""
import enlighten
with enlighten.Counter(total=1, desc='A{') as pbar:
    pass
```

```python
"""
Raw counter
"""
import enlighten
with enlighten.Counter(unit='B}') as pbar:
    pass
```

**Environment (please complete the following information):**
 - Enlighten Version: 1.10.1
 - OS and version: Windows 10
 - Console application: Cmd/PowerShell/Pycharm
 - Special Conditions: No

**Additional context**
N/A

**Comment by avylove** [comment]
This is because Enlighten uses str.format() internally to do variable substitution. To escape a curly brace, you need to double it.

```
with enlighten.Counter(total=1, desc='A{{') as pbar:
    pass
```

Let me know if that resolves your issue.
This should probably be in the FAQ.

**Comment by ZivRonen** [comment]
Yes. I just thought it can be solved in the library, or at least be documented (if it was, I missed it).

Note: In my case the values are generated dynamically (os.walk of an unknown client) so I used:
```python
with enlighten.Counter(total=1, desc=path.replace('{', '{{').replace('}', '}}') as pbar:
   pass
```

which solved the issue for me.

Thanks for the quick response

**Comment by avylove** [comment]
I just pushed a commit to take care of it. Let me know if it works for you. It would probably still be a good idea to add tests to protect against regression.

**Comment by ZivRonen** [comment]
Checked it on my sample and it worked great!
Also checked the code and it look fine to me, I don't see a real case that someone will use the new place holder by mistake :)

**Comment by avylove** [comment]
Released in [1.10.2](https://github.com/Rockhopper-Technologies/enlighten/releases/tag/1.10.2)

**Comment by ZivRonen** [comment]
Thank you!
